clients/src/main/java/org/apache/kafka/clients/admin/OffsetSpec.java

```diff
@@ -33,7 +33,7 @@ public static class TimestampSpec extends OffsetSpec {
             this.timestamp = timestamp;
         }
 
-        long timestamp() {
+        public long timestamp() {
```

Comment:
We need to maintain it in the future if we change it public and may bring some compatible problem, we can move `getOffsetFromSpec` to `KafkaAdminClient` to solve it.